sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/ShuffleExchangeExec.scala

```diff
@@ -264,7 +264,11 @@ object ShuffleExchangeExec {
             prefixComparator,
             prefixComputer,
             pageSize,
-            canUseRadixSort)
+            // As we need to compare the binary of UnsafeRow here, we can't make sure whether all
+            // the fields can sort fully with prefix like SortExec. So we disable radix sort here
+            // to avoid getting unstable sort, and result to a correctness bug.
+            // See more details in SPARK-28699.
+            false /* canUseRadixSort */)
```

Comment:
No more canUseRadixSort here, you can just remove /* canUseRadixSort */?

Reply:
Sure, done in next commit.